editor/src/clj/editor/app_view.clj

```diff
@@ -2012,6 +2012,8 @@ If you do not specifically require different script states, consider changing th
               :header (format "Unable to open '%s', since it contains unrecognizable data. Could the project be missing a required extension?" (resource/proj-path resource))})
            false)
        (if-let [custom-editor (and (#{:code :text} (:id view-type))
+                                   (or (:selected-view-type opts)
+                                       (get-in resource-type [:view-opts (:id view-type) :use-custom-editor] true))
```

Comment:
`(:selected-view-type opts)` will have a `view-type` map if the user used **Open As...** to explicitly specify how to open the file. It will be `nil` if they used the default **Open** command, or double-clicked a file.